### DISCUSSION TITLE
pythonnet can't import EPPlus

### DISCUSSION THREAD

**Discussion by retsyo** [pythonnet/pythonnet#1949] [Q&A]
First of all, I know that there are python packages, for example openpyxl, can be used to read/write Excel XLSX file without MS office installation. However in my case I try to use https://github.com/EPPlusSoftware/EPPlus via https://github.com/pythonnet. Here is my try in python 3.8.1 64 bits on windows 10 64 bits.

I have pythonnet-2.5.2 via `pip`.
```bash
>>> import sys
>>> sys.path.append('.')
>>>
>>> import clr
>>>
>>> clr.AddReference('EPPlus')
<System.Reflection.RuntimeAssembly object at 0x000001647A1ED7F0>
```
Then I can't import module in `EPPlus.dll`
```
>>> import EPPlus
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'EPPlus'

>>> import OfficeOpenXml
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'OfficeOpenXml'
```

So what is the proper way to use `EPPlus.dll` in pythonnet? Thanks

**Comment by lostmsu** [comment]
Please, try 3.0.0 preview from `pip`

**Comment by retsyo** [comment]
thanks. but is there any pre-built pythonnet 3.0 binary for pip? There is no C#/C++ compiler installation on my PC.

**Reply by filmor** [reply]
If you'd tried it you'd knew :). Hint: https://pypi.org/project/pythonnet/3.0.0rc6/#files
